django_netjsongraph/migrations/0011_move_to_key_field.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.9 on 2020-01-17 15:32
+
+import re
+
+import django.core.validators
+import openwisp_utils.base
+import openwisp_utils.utils
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_netjsongraph', '0010_remove_old_addresses'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='topology',
+            name='key',
+            field=openwisp_utils.base.KeyField(blank=True, default=openwisp_utils.utils.get_random_key, help_text='key needed to update topology from nodes ', max_length=64, validators=[django.core.validators.RegexValidator(re.compile('^[^\\s/\\.]+$'), code='invalid', message='This value must not contain spaces, dots or slashes.')], verbose_name='key'),
+        ),
+    ]
```

Comment:
this migration file should not be needed now that you updated the old migration file, you can remove it

Reply:
@nemesisdesign I updated old migration file, so that it's old generator (which was in `django-netjsongraph.utils`) wouldn't be needed, as I deleted it from there. But in the new migration I change the entire field type. So it seems to me, that we still need new migration file. I can of course update the old file once again to make it also `KeyField`, but I think it's better the way it is.

Reply:
change the entire field in the old migration file